lib/is-prerelease.js

```diff
@@ -1,3 +1,11 @@
-export default function isPrerelease({ type, main }) {
-  return type === "prerelease" || (type === "release" && !main);
+export default function isPrerelease({ type, main, prerelease }) {
+  if (typeof prerelease === "boolean" && prerelease === false) {
```

Comment:
This is the same as

```suggestion
  if (prerelease === false) {
```

The triple equal operator checks if the variables are equal in both value _and_ type